qDict/src/main/res/layout/about.xml

```diff
@@ -1,59 +1,72 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout android:id="@+id/root_about"
-                xmlns:android="http://schemas.android.com/apk/res/android"
-                android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:background="@color/mmt_white"
-                android:paddingBottom="15dp">
-
-    <ImageView
-        android:id="@+id/iv_icon"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginLeft="@dimen/about_padding"
-        android:layout_marginTop="@dimen/large_padding"
-        android:contentDescription="@string/app_name"
-        android:src="@drawable/ic_launcher">
-    </ImageView>
-
-    <com.mmt.widget.QTextView
-        android:id="@+id/tv_app_name"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_alignTop="@id/iv_icon"
-        android:layout_marginLeft="18dp"
-        android:layout_marginTop="2dp"
-        android:layout_toRightOf="@id/iv_icon"
-        android:text="@string/app_name"
-        android:textAppearance="?android:attr/textAppearanceLarge"
-        android:textColor="@color/mmt_black"/>
-
-    <com.mmt.widget.QTextView
-        android:id="@+id/tv_app_version_label"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_alignLeft="@id/tv_app_name"
-        android:layout_below="@id/tv_app_name"
-        android:text="@string/app_version_label"
-        android:textAppearance="?android:attr/textAppearanceSmall"
-        android:textColor="@color/mmt_black"/>
-
-    <com.mmt.widget.QTextView
-        android:id="@+id/tv_app_version"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_below="@id/tv_app_name"
-        android:layout_marginLeft="5dip"
-        android:layout_toRightOf="@id/tv_app_version_label"
-        android:text="@string/app_version"
-        android:textAppearance="?android:attr/textAppearanceSmall"
-        android:textColor="@color/mmt_black"/>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/root_about"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/mmt_white"
+    android:paddingBottom="15dp">
+
+    <LinearLayout
+        android:id="@+id/icon_name_constraint_layout"
```

Comment:
nitpick: it's no longer a constraint_layout.

Reply:
OK, thanks for reminding. :)